src/comp_loinc/__init__.py

```diff
@@ -1,2 +1,7 @@
-from .generator import Generator
-from .cli import cli
+"""Computational LOINC"""
+from comp_loinc.cli import cli
+from comp_loinc.generator import Generator
+from comp_loinc.main import app
+
+# cli()   # not callable. what method should be called?
+app()
```

Comment:
## `__main__.py` & `__init__.py` updates
I deleted `__main__.py`. We can do what we need with `__init__.py`, though some add a `__main__.py` to draw some distinctions between things that are just being imported in the top level name space, vs what is supposed to happen when you run `python -m MY.PACKAGE.PATH`. But I like having 1, especially since we already have a file called `main`, which makes having 2 mains a little confusing.

I was having some problems with the relative imports. This happens to me at times, and I'm embarrassed to say that I have done a lot of reading and it's not entirely clear to me how to get it to consistently work. But these namespaced imports work for me whether running the command line, or running the PyCharm debugger (with script set to `src/comp_loinc/__init__.py` and working dir set to `src/comp_loinc`. 

@ShahimEssaid Let me know if this causes any issues for you and I can tweak the setup slightly to work for both of us, or I can look at your dev environment and maybe copy what you have.

Reply:
Actually I changed this setup a few times before deciding I liked it closer to how you had it before. Never mind.